artemis/src/main/java/tech/pegasys/artemis/cli/deposit/CommonParams.java

```diff
@@ -154,6 +154,10 @@ public UnsignedLong getAmount() {
     return amount;
   }
 
+  public boolean isDisplayConfirmation() {
+    return displayConfirmation;
+  }
```

Comment:
We should only display the confirmation when we're outputting to a console.  So when you run the command with output redirected to a file it should automatically not show the warning (which will corrupt your file).  Not sure if it will help with the docker tests but would definitely be a good thing to do for normal command line users.

Reply:
yes, we only display confirmation if we are not outputting to a file. I exposed this method so that we can suppress the warning when we are in "console" mode so that we don't have to parse the output for acceptance test.

Reply:
I think you're misunderstanding.  Even when you're outputting to system out you might be outputting to a file.  e.g.
`teku validator generate > someFile.yaml`

We can check if `System.console()` returns null to set the default value (it's not perfect but seems to be the best you can do from plain java).

Reply:
Ohh ... we are currently throwing error if System.console returned null, I need to slightly re-design the approach.

Reply:
we have introduced "interactive mode" ... for which we check System.console not to be null.

Reply:
Console can be null if either stdin or stdout is null :(. We only want to supress if stdout is null :(

Reply:
http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/solaris/native/java/io/Console_md.c#l36

Reply:
If stdin is null someone is doing something interesting so you may as well just suppress.  It's better to suppress the message than to accidentally create a corrupt file.

Reply:
In case of stdout redirection we don't print out warnings anymore.